packages/graphql/src/app.ts

```diff
@@ -44,7 +42,7 @@ if (process.env.NODE_ENV !== 'test') {
 app.use(convert(cors({maxAge: 86400, origin: '*'})));
 
 app.use(async (ctx, next) => {
-  ctx.dataloaders = getDataLoaders(loaders);
+  ctx.container = container.cradle;
```

Comment:
Why are you removing `dataloaders` from Koa context?

Reply:
`dataloaders` are database related so I plan on moving them to their respective repositories

Reply:
Which repository?

Reply:
DDD Repository layer.

Reply:
@belgamo could you provide some links about that architecture? I'd like to know some more.